### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -L
ans,wth,nd,ist,dout,coo,childs,nin,ded,ons,nam,perfor,malins,stoer,mis,abritrary,commun,ser,te,equil,numer,thi,clos,infor,hax,pich,inout,alledges`

### CODE REVIEW

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ You can report issues to the
 at networkx-conduct@groups.io.
 
 If your report involves any members of the Council, or if they feel they have
-a conflict of interest in handling it, then they will recuse themselves from
+a conflict of interest in handling it, then they will recurse themselves from
```
Comment (jarrodmillman) [inline CODE_OF_CONDUCT.rst:116]:
```suggestion
a conflict of interest in handling it, then they will recuse themselves from
```

This was correct.

File: examples/geospatial/schools.shp.xml [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
...4</southBL><exTypeCode Sync="TRUE">1</exTypeCode></GeoBndBox></geoEle></dataExt><suppInfo>This data set does not contain locations for all cosmetology or beauty colleges, horseshoeing or welding technical schools, or other trade schools.</suppInfo><envirDesc Sync="TRUE">Microsoft Windows XP Version 5.1 (Build 2600) Service Pack 3; ESRI ArcGIS 10.0.2.3200</envirDesc><descKeys><thesaName uuidref="723f6998-058e-11dc-8314-0800200c9a66"></thesaName><keyword Sync="TRUE">002</keyword></descKeys><spatRpType><SpatRepTypCd value="001"></SpatRepTypCd></spatRpType></dataIdInfo><dqInfo><dqScope><scpLvl><ScopeCd value="005"/></scpLvl></dqScope><dataLineage/></dqInfo><spatRepInfo><VectSpatRep><topLvl><TopoLevCd value="001"></TopoLevCd></topLvl><geometObjs><geoObjTyp><GeoObjTypCd value="004"></GeoObjTypCd><geoObjCnt Sync="TRUE">2772</geoObjCnt></geoObjTyp></geometObjs></VectSpatRep></spatRepInfo><Binary><Enclosure><Descript>original metadata</Descript><Data SourceMetadata="yes" OriginalFileName="source_metadata.xml" SourceMetadataDigest="3bf31794c3811024c8bd56a81c2e3c" EsriPropertyType="Base64" SourceMetadataSchema="fgdc">PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjxtZXRhZGF0YT4NCiAgPGlk
```
Comment (jarrodmillman) [inline examples/geospatial/schools.shp.xml:6]: I don't not what changed. But this is a dataset, so we should leave it alone.

**Review by jarrodmillman — CHANGES_REQUESTED** [review]
Once those two changes are reverted, this should be good to go. Thanks!!

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**